harc_game_web/apps/posts/views.py

```diff
@@ -0,0 +1,92 @@
+from django.shortcuts import render, redirect, get_object_or_404, reverse
+from django import forms
+from .models import Post
+from django.http import HttpResponseRedirect
+from django.utils import timezone
+from django.db.models import F
+
+class PostForm(forms.ModelForm):
+    class Meta:
+        model = Post
+        fields = ['pub_date_time', 'title', 'content', 'link1', 'link2', 'link3']
+  
+def ListActivePosts(request):
```

Comment:
following PEP8, names of functions and methods should be lowercase with words separated by underscore (e.g. `list_active_posts`. Please fix it here and in other places 🙏

Reply:
Ok... I saw you used mixed mode and did not know what is preference. Will do

Reply:
Or maybe not...